common/character_interactions/00_alliance.txt

```diff
@@ -106,8 +106,10 @@
 				text = can_call_in_civil_war_trigger_loc
```

Comment:
`_mapFiller/provinceDef.xls` is deleted.

Reply:
I believe this is a bug in GitHub Desktop. It showed the file as deleted in the UI after I editted it, but it's not. Refer to the comment after the next on associated changes to the xls.

Reply:
It's on GitHud.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/162030232-8ec723a2-6566-4043-b7f7-cd7e127d7735.png)

</details>

Mkay, but I would keep this weirness in mind.

Reply:
I'll request testing before merging just to be safe.